chart/templates/NOTES.txt

```diff
@@ -30,10 +33,26 @@ Flower dashboard:      http{{ if .Values.ingress.flower.tls.enabled }}s{{ end }}
 {{- else }}
 You can now access your dashboard(s) by executing the following command(s) and visiting the corresponding port at localhost in your browser:
 
-Airflow dashboard:        kubectl port-forward svc/{{ .Release.Name }}-webserver {{ .Values.ports.airflowUI }}:{{ .Values.ports.airflowUI }} --namespace {{ .Release.Namespace }}
+Airflow Webserver:      kubectl port-forward svc/{{ .Release.Name }}-webserver {{ .Values.ports.airflowUI }}:{{ .Values.ports.airflowUI }} --namespace {{ .Release.Namespace }}
+
 {{- if .Values.flower.enabled }}
 {{- if or (eq .Values.executor "CeleryExecutor") (eq .Values.executor "CeleryKubernetesExecutor")}}
 Flower dashboard:         kubectl port-forward svc/{{ .Release.Name }}-flower {{ .Values.ports.flowerUI }}:{{ .Values.ports.flowerUI }} --namespace {{ .Release.Namespace }}
 {{- end }}
 {{- end }}
 {{- end }}
+
+
+{{- if .Values.webserver.defaultUser.enabled}}
+Default Webserver (Airflow UI) Login credentials:
+    username: {{ .Values.webserver.defaultUser.username }}
+    password: {{ .Values.webserver.defaultUser.password }}
+{{- end }}
+
+{{- if .Values.postgresql.enabled }}
+Default Postgres connection credentials:
+    username: {{ .Values.data.metadataConnection.user }}
+    password: {{ .Values.data.metadataConnection.pass }}
+    port: {{ .Values.data.metadataConnection.port }}
+
+{{- end }}
```

Comment:
```suggestion
{{- end }}

{{- if not .Values.fernetKeySecretName }}

You can get Fernet Key value by running the following:

    echo Fernet Key: $(kubectl get secret --namespace {{ .Release.Namespace }} {{ .Release.Name }}-fernet-key -o jsonpath="{.data.fernet-key}" | base64 --decode)

{{- end }}
```

Reply:
This will add instruction on how to get Fernet Key

Reply:
Example:

![image](https://user-images.githubusercontent.com/8811558/118136486-13306000-b3fc-11eb-94e9-d55465444f31.png)